.semaphore/daily-builds.yml

```diff
@@ -3262,6 +3262,7 @@ blocks:
             - make check.ruby.code
         - name: "\U0001F6E1️ Check dependencies"
           commands:
+            - sem-version ruby 3.3
```

Comment:
On line 27 we are setting ruby to 3.0 for toolbox. Instead of changing it to 3.3 just here, please do it there for the whole toolbox.